runtime/runtime-params-estimator/src/trie.rs

```diff
@@ -0,0 +1,417 @@
+use crate::estimator_context::Testbed;
+use crate::gas_cost::{GasCost, NonNegativeTolerance};
+use crate::utils::{aggregate_per_block_measurements, percentiles};
+use near_primitives::hash::hash;
+use near_primitives::types::TrieCacheMode;
+use near_store::{TrieCachingStorage, TrieStorage};
+use near_vm_logic::ExtCosts;
+use std::iter;
+use std::sync::atomic::{AtomicUsize, Ordering};
+
+static SINK: AtomicUsize = AtomicUsize::new(0);
+
+pub(crate) fn write_node(
+    testbed: &mut Testbed,
+    warmup_iters: usize,
+    measured_iters: usize,
+    final_key_len: usize,
+) -> GasCost {
+    let tb = testbed.transaction_builder();
+    // Prepare a long chain in the trie
+    let signer = tb.random_account();
+    let key = std::iter::repeat('j').take(final_key_len).collect::<String>();
+    let mut setup_block = Vec::new();
+    for key_len in 0..final_key_len {
+        let key = &key.as_str()[..key_len].as_bytes();
+        let value = "0";
```

Comment:
`let value = b"0";` and you won’t need `.as_bytes()`.  Similarly below with `[b"1", b"2", b"3"]`.  Same applies to read_node_from_db.